test/featureflags.go

```diff
@@ -117,6 +117,7 @@ func getFeatureFlagsBaseOnAPIFlag(t *testing.T) *config.FeatureFlags {
 		"enable-step-actions":          "true",
 		"enable-cel-in-whenexpression": "true",
 		"enable-param-enum":            "true",
+		"enable-artifacts":             "true",
```

Comment:
Thanks for updating this, would you mind also adding it to https://github.com/tektoncd/pipeline/blob/a9d8a9b1c7459b64190bd0f73014cee6e56e2df1/test/per_feature_flags_test.go#L50?

Reply:
thanks, added